documentation/rosetta-modelling-component.md

```diff
@@ -1580,33 +1594,35 @@ Hints are synonyms used to bypass a layer of rosetta without *consuming* an inpu
 e.g. :
 
 ```
-ResolvablePayoutQuantity:
-  + assetIdentifier
-    [value "notionalAmount"]
-    [hint "currency"]
+type EngineSpecification:
+    engineMetric EngineMetric (0..*)
+        [synonym MULTI_CARDINALITY_EXAMPLE_12 value "capacityDetail"]
+        [synonym MULTI_CARDINALITY_EXAMPLE_12 hint "combustible"]
+
+
+type EngineMetric:
+    fuel string (0..1)
+        [synonym MULTI_CARDINALITY_EXAMPLE_12 value "combustible"]
 
-AssetIdentifier:
-  + currency
-    [value "currency" maps 2 meta "currencyScheme"]
 ```
 
-In this example the input attribute \"notionalAmount\" is matched to the assetIdentifier and the children of \"notionalAmount\" will be matched against the synonyms for AssetIdentifier. However the input attribute \"currency\" will also be matched to the assetIdentifier but \"currency\" is still available to be matched against the synonyms of AssetIdentifier.
+In this example the input attribute \"capacityDetail\" is matched to the engineMetric and the children of \"capacityDetail\" will be matched against the synonyms for EngineMetric. However the input attribute \"combustible\" will also be matched to the engineMetric but \"combustible\" is still available to be matched against the synonyms of EngineMetric.
 
 #### Merging inputs
 
 Where a Rosetta attribute exists with multiple cardinality, to which more than one input element maps, synonyms can be used to either create a single instance of the Rosetta attribute that merges the input elements or to create multiple attributes - one for each input element. E.g. The synonyms :
 
 ```
-interestRatePayout InterestRatePayout (0..*)
-  [synonym FpML_5_10 value feeLeg]
-  [synonym FpML_5_10 value generalTerms]
+engineSpecification EngineSpecification (0..*)
+  [synonym MULTI_CARDINALITY_EXAMPLE_20 value "fuelDetail"]
+  [synonym MULTI_CARDINALITY_EXAMPLE_20 value "capacityDetail"]
 ```
 
-will produce two InterestRatePayout objects. In order to create a single InterestRatePayout with values from the FpML `feeLeg` and `generalTerms` the synonym merging syntax should be used:
+will produce two EngineSpecification objects. In order to create a single EngineSpecification with values from the FpML `fuelDetail ` and `capacityDetail` the synonym merging syntax should be used:
```

Comment:
Here you are mentioning "FpML". Remove or replace by "from the attributes" or "inputs"?

Reply:
okey, will do that change.